src/components/Checkout/BraintreeCheckout.vue

```diff
@@ -104,11 +104,11 @@
 					<div id="kv-expiration-date" class="kv-braintree-wrapper"></div>
 				</div>
 				<div class="small-4 columns">
```

Comment:
Sneaking this in, not related to KvPillToggle, just noticed it on the same page.